File: JAVA_SIS104/app/src/main/java/com/example/prueba_java/backend/WSacciones.java
package com.example.prueba_java.backend;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WSacciones {
        @SerializedName("id")
        @Expose
        private Integer id;
        @SerializedName("accion")
        @Expose
        private String accion;
        @SerializedName("descripcion")
        @Expose
        private String descripcion;
        @SerializedName("fecha")
        @Expose
        private String fecha;

        public Integer getId() {
            return id;
        }

        public void setId(Integer id) {
            this.id = id;
        }

        public String getAccion() {
            return accion;
        }

        public void setAccion(String accion) {
            this.accion = accion;
        }

        public String getDescripcion() {
            return descripcion;
        }

        public void setDescripcion(String descripcion) {
            this.descripcion = descripcion;
        }

        public String getFecha() {
            return fecha;
        }

        public void setFecha(String fecha) {
            this.fecha = fecha;
        }

}

File: KOTLIN_SIS104/app/src/main/java/com/example/prueba_kotlin/backend/WSAcciones.kt
package com.example.prueba_kotlin.backend

class WSAcciones {
    @com.google.gson.annotations.SerializedName("id")
    @com.google.gson.annotations.Expose
    private var id: Int? = null

    @com.google.gson.annotations.SerializedName("accion")
    @com.google.gson.annotations.Expose
    private var accion: String? = null

    @com.google.gson.annotations.SerializedName("descripcion")
    @com.google.gson.annotations.Expose
    private var descripcion: String? = null

    @com.google.gson.annotations.SerializedName("fecha")
    @com.google.gson.annotations.Expose
    private var fecha: String? = null

    fun getId(): Int? {
        return id
    }

    fun setId(id: Int?) {
        this.id = id
    }

    fun getAccion(): String? {
        return accion
    }

    fun setAccion(accion: String?) {
        this.accion = accion
    }

    fun getDescripcion(): String? {
        return descripcion
    }

    fun setDescripcion(descripcion: String?) {
        this.descripcion = descripcion
    }

    fun getFecha(): String? {
        return fecha
    }

    fun setFecha(fecha: String?) {
        this.fecha = fecha
    }
}